scripts/quests/bastok/Ayame_and_Kaede.lua

```diff
@@ -0,0 +1,214 @@
+-----------------------------------
+-- Ayame and Kaede
+-----------------------------------
+-- Log ID: 1, Quest ID: 60
+-- Kaede    : !pos 48 -6 67 236
+-- Kagetora : !pos -96 -2 29 236
+-- Ensetsu  : !pos 33 -6 67 236
+-- qm2      : !pos -208 -9 176 173
+-- Ryoma    : !pos -23 0 -9 252
+-----------------------------------
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/settings')
+require('scripts/globals/status')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local korrolokaID  = require('scripts/zones/Korroloka_Tunnel/IDs')
+local portBastokID = require('scripts/zones/Port_Bastok/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.AYAME_AND_KAEDE)
+
+quest.reward =
+{
+    fame     = 30,
+    fameArea = xi.quest.fame_area.BASTOK,
+    title    = xi.title.SHADOW_WALKER,
+}
+
+local function isNMSpawned()
+    for nmId = korrolokaID.mob.KORROLOKA_LEECH, korrolokaID.mob.KORROLOKA_LEECH + 2 do
+        if GetMobByID(nmId):isSpawned() then
+            return true
+        end
+    end
+
+    return false
+end
+
+local function isNMDefeated()
+    for nmId = korrolokaID.mob.KORROLOKA_LEECH, korrolokaID.mob.KORROLOKA_LEECH + 2 do
+        if GetMobByID(nmId):isDead() then
+            return true
+        end
+    end
+
+    return false
+end
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainLvl() >= xi.settings.main.ADVANCED_JOB_LEVEL
+        end,
+
+        [xi.zone.PORT_BASTOK] =
+        {
+            ['Kaede'] = quest:progressEvent(240),
+
+            onEventFinish =
+            {
+                [240] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.KORROLOKA_TUNNEL] =
+        {
+            ['qm2'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        not isNMSpawned() and
+                        quest:getVar(player, 'Prog') == 2
+                    then
+                        if quest:getLocalVar(player, 'nmDefeated') == 1 then
+                            quest:setVar(player, 'Prog', 3)
+
+                            if quest:getLocalVar(player, 'isActor') == 1 then
+                                quest:getLocalVar(player, 'isActor', 0)
```

Comment:
set
`quest:setLocalVar(player, 'isActor', 0)`

Reply:
And this is what I get for making changes before coffee